PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsDotNetCoreIsolatedDebugProfile.kt

```diff
@@ -66,26 +74,30 @@ class AzureFunctionsDotNetCoreIsolatedDebugProfile(
         // Launch Azure Functions host process
         launchAzureFunctionsHost()
 
+        // Show progress bar
+        ProgressManager.getInstance().run(
```

Comment:
This construction looks pretty unusual from my view - you run a courting that start a fake background activity just to show a progress. Why not to put all your logic inside this background activity, so, you do not need to handle progress separately from the actual activity you do?

Reply:
Agree. Do you know if we can block execution of the `createWorkerRunInfo` method in this background activity? (like `pumpMessages` does now)